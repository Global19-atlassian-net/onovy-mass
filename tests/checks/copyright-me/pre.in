#!/bin/bash

# Git
git init >/dev/null
git commit --author "$DEBFULLNAME <$DEBEMAIL>" -m "Dummy commit" --allow-empty >/dev/null

# Current year
CURRENT_YEAR=$(date +%Y)
sed -i "s/#CURRENT_YEAR#/$CURRENT_YEAR/g" debian/copyright
