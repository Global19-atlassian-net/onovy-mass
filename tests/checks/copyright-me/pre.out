#!/bin/bash

# Current year
CURRENT_YEAR=$(date +%Y)
sed -i "s/#CURRENT_YEAR#/$CURRENT_YEAR/g" debian/copyright
